frontend/app/_components/query-abstract-subgraph.tsx

```diff
@@ -0,0 +1,73 @@
+'use client'
+
+import { useAccount } from "graz"
+import { useAccountsMetadataGraphQLQuery } from "../_hooks/useQueryAccountsById"
+import { useAccounts } from "@abstract-money/react"
+import { appChain } from "../../utils/chains"
+import { Card, CardContent, CardHeader, CardTitle } from "../../components/card"
+import { Alert, AlertDescription, AlertTitle } from "../../components/alert"
+import { AlertCircle } from "lucide-react"
+
+export const QueryAbstractSubgraph: React.FC = () => {
+  const chainId = appChain.chainId
+
+  const { data: cosmosAccount } = useAccount({ chainId })
+  const { data: accounts } = useAccounts({
+    args: {
+      owner: cosmosAccount?.bech32Address ?? "",
+      chains: [appChain.chainName],
+    },
+    query: {
+      enabled: !!cosmosAccount?.bech32Address,
+    },
+  })
+  const { data: accountsMetadata, isLoading } = useAccountsMetadataGraphQLQuery({ accountIds: accounts })
+
+  return (
+    <Card>
+      <CardHeader>
+        <CardTitle>Query Abstract Subgraph Example</CardTitle>
+      </CardHeader>
+      <CardContent>
+        <div className="flex flex-col gap-4">
+          {isLoading && <p>Loading account details...</p>}
+          {accountsMetadata && accountsMetadata.length > 0 ? (
+            <div className="bg-gray-100 p-3 rounded-md">
+              <h3 className="font-semibold mb-2">Account Information:</h3>
+              {accountsMetadata.map((account) => (
+                <div key={account.id} className="mb-4">
+                  <p>
+                    <strong>ID:</strong> {account.id}
+                  </p>
+                  <p>
+                    <strong>Name:</strong> {account.info.name}
+                  </p>
+                  <p>
+                    <strong>Chain ID:</strong> {account.info.chainId}
+                  </p>
+                  <p>
+                    <strong>Proxy:</strong> {account.proxy}
+                  </p>
+                  <p>
+                    <strong>Manager:</strong> {account.manager}
+                  </p>
```

Comment:
This will change with Abstract 0.24
```suggestion
                  <p>
                    <strong>Account:</strong> {account.address}
                  </p>
```